package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "prism-documentation-portal",
+  "name": "identus-documentation-portal",
```

Comment:
Is this file not generated automatically by the workflow (maybe this .github/workflows/build-docs.yml)? So I would not change this file.